sections/main-article.liquid

```diff
@@ -25,6 +25,7 @@
                 sizes="(min-width: {{ settings.page_width }}px) {{ settings.page_width | minus: 100 }}px, (min-width: 750px) calc(100vw - 10rem), 100vw"
                 src="{{ article.image | image_url: width: 1100 }}"
                 loading="eager"
+                fetchpriority="high"
```

Comment:
I like the plus of having this and how beneficial it can be but I'm not too sure where I stand on using experimental technology on our theme. 
This is something we've been trying to avoid where we can to make sure we're limiting our need to add code that will need to be addressed in the future once it's fully supported or if the implementation/specs change. 

This is supported only in the latest version of Chrome and Edge ([MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority)). 

Do you feel confident that the implementation is going to stay that way ? Experimental sound like it's possible for it to change but maybe in this case it's just because of the low support. 
![](https://screenshot.click/24-51-za15p-k3yfq.png)

Reply:
Very few features have 100% browser support, and this one has no negative effect on those other browsers. By implementing it, we can make the site significantly faster for all Chrome, Edge, and Opera users which is a pretty large bucket - 67.91% of users globally. If in the rare case it's pulled, then functionally it will not break the code. The browsers that do not recognize it will ignore the attribute.

Reply:
FYI, Safari has it in Technology Preview per caniuse

Reply:
I pinged the Chrome engineer who worked on it to learn more - it's stable in Chrome so really just waiting for another browser to implement to remove the "experimental" flag. The other browsers already participated in the naming. So it's low risk to implement with big gains for Chromium performance:
<img width="698" alt="image" src="https://user-images.githubusercontent.com/4777393/193314112-bdf381f5-4187-4786-9307-38a99a2a80a2.png">